src/three-idx.ts

```diff
@@ -209,7 +211,7 @@ export class ThreeIDX {
     promises.push(
       this.docs.idx.change({
         content: Object.assign(this.docs.idx.content, {
-          [KEYCHAIN_DEF]: this.docs[KEYCHAIN_DEF].id,
+          [KEYCHAIN_DEF]: this.docs[KEYCHAIN_DEF].id.toUrl('base36'),
```

Comment:
Where we not changing this to not be the url?

Reply:
I changed the definitions to only use the string representation of the DocID, but otherwise I kept using URLs so hopefully the JSON schema validation can help ensure only DocIDs are used as references.

Reply:
Wait, so are we planning to use urls or not now? I'm confused.

Reply:
Yes I kept URLs for now. We can change this if we want but then we'll need to update all the schemas using the URL format for docIDs.